server/utils/trycatch.helper.js

```diff
@@ -0,0 +1,18 @@
+/**
```

Comment:
@cupidcoder I'd like to know why this helper function exists and what it solves for?

Reply:
@waleCloud the team agreed to use a try-catch wrapper that wraps all controller methods. This way, there wouldn't be repeated blocks of try-catch code.

Reply:
Okay cool.
I realized there's a status code of `50X` associated with the try catch.
My question is throwing a `50X` all the time the application fails. will be descriptive enough?
how are you handling a `40X` or related errors?

Reply:
We assumed that `40X` errors would be related to input validation errors which would be caught in the` middleware` validation functions @davidokonji would be writing.